rb/Gemfile

```diff
@@ -5,5 +5,6 @@ Dir["#{__dir__}/*.gemspec"].each do |spec|
   gemspec name: File.basename(spec, '.gemspec')
 end
 
+gem 'curb', '~> 1.0.5', require: false, platforms: %i[darwin]
```

Comment:
Why only Darwin? Curb should at least install on Linux/macOS. For Windows, I think it should be possible to install https://packages.msys2.org/package/libcurl?repo=msys&variant=x86_64 for curb to work.

Reply:
@p0deje I want to see if all the pipelines failed again, because the build on ubuntu was failing due to curb, btw this line of code doesn't work neither install_if: darwin so a simple conditional will make the trick

Reply:
You might need to install `libcurl` with GitHub Actions before running the build.

Reply:
Linux worked, now I'm trying on windows

Reply:
For Windows, since Ruby is installed via Bazel, you might want to instruct it to install libcurl too, this is done via 

```bzl
# MODULE.bazel
ruby.toolchain(
    name = "ruby",
    version_file = "//:.ruby-version",
    msys2_packages = ["libcurl", "libyaml"]
)
```

Reply:
I will give it a try to this :) thank you

Reply:
I changed the package manager on github action to msys2, so hopefully this will help, it said before with bazel that it couldn't find the package

Reply:
When Bazel installs Ruby on Windows, it would actually ignore system-installed MSYS2 so it would help.

I think libcurl is a wrong name, looking at https://packages.msys2.org/base/mingw-w64-curl, you probably can try install `curl` and see if it helps - Bazel will add `mingw-w64-x86_64-` prefix automatically.